src/app/@ansyn/plugins/openlayers/maps/open-layers-map/openlayers-map/openlayers-map.ts

```diff
@@ -104,29 +110,29 @@ export class OpenLayersMap extends BaseImageryMap<OLMap> {
 			loadTilesWhileInteracting: true,
 			loadTilesWhileAnimating: true
 		});
-		this.initListeners();
 		return this.resetView(layers[0], position);
 	}
 
 	initListeners() {
-		this._moveEndListener = () => {
-			this.getPosition().pipe(take(1)).subscribe(position => {
-				if (position) {
-					this.positionChanged.emit(position);
-				}
-			});
-		};
-
 		this._mapObject.on('moveend', this._moveEndListener);
 	}
 
+	disposeListeners(): void {
+		this._mapObject.un('moveend', this._moveEndListener);
+	}
+
 	createView(layer): View {
 		return new View({
 			projection: layer.getSource().getProjection()
 		});
 	}
 
-	public resetView(layer: any, position: ICaseMapPosition, extent?: CaseMapExtent): Observable<boolean> {
+	public resetView(layer: any, position: ICaseMapPosition, extent?: CaseMapExtent, notGeoRegistred?: boolean): Observable<boolean> {
+		this.disposeListeners();
+		if (!notGeoRegistred) {
+			this.initListeners();
```

Comment:
need to prevent all get/set position, center... operations thats uses projections